linalg_ops/test_utils.c

```diff
@@ -22,7 +22,7 @@
 #include "iree/tooling/device_util.h"
 #include "iree/vm/api.h"
 
-IREE_FLAG(bool, require_exact_results, true,
+IREE_FLAG(bool, require_exact_results, false,
```

Comment:
You could set this flag value per test, rather than flip the default. I don't have that strong an opinion on it.